command/deploy.go

```diff
@@ -68,6 +68,9 @@ General Options:
   -var-file=<file>
     Used in conjunction with the -job-file will deploy a templated job to your
     Nomad cluster. [default: levant.(yaml|yml|tf)]
+
+  -allow-stale
```

Comment:
pedantic but could we move this up and keep the flags list in alphabetical order?